docs/reference/setup/install/docker.asciidoc

```diff
@@ -48,7 +48,7 @@ Elasticsearch can be quickly started for development or testing use with the fol
 
 ["source","sh",subs="attributes"]
 --------------------------------------------
-docker run -p 9200:9200 -e "http.host=0.0.0.0" -e "transport.host=127.0.0.1" {docker-image}
+docker run -p 9200:9200 {docker-image} -E "http.bind_host=0.0.0.0" -E "http.publish_host=127.0.0.1" -E "transport.host=127.0.0.1"
```

Comment:
The provided example did not seems to work in my case, but I was using Docker for windows. After hours of trying to understand what was going on, this is the only thing that made it working (and it's working ever since)